Sparkbox sprint 51

pebutler3 opened the pull request:
# READY FOR REVIEW

## Summary
- #979 
- #980 
- #977 
- #973 
- #916 
- #983 

### TASK LIST:
- [x] Add compiled CSS & JS
- [x] Run BackstopJS visual regression tests
- [x] Ensure Code Climate passes against the `develop` branch. Link here: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/976
- [x] Assign to Mike for review.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. `npm run test` should pass in the root folder
2. If you are interested in the testing steps for associated PRs, please take a look at the individual PR's description and "Steps to Test" section.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

Commit:
Feat(STN-771): vertical timeline raised cards (#977)

* feat(STN-771): Timeline Refinement (Vertical) --> Add hb-raised-cards styles

* docs: update compatibility table

* style: lint errors

* chore: compile css

Co-authored-by: Ryan Kelbel <kelbel@heysparkbox.com>

Commit:
STN-723:  Add documentation on configuration deploys (#916)

* feat (STN-723): Add documenatation for deploying new components without impacting legacy and existing sites

* feat: add description and listing of legacy and current site themes

* feat: update documentation readme by adding link to new component deployment doc

* feat: add post update functions description

* Add examples and descriptions to the config override deployment documentation

Co-authored-by: Ryan Kelbel <kelbel@heysparkbox.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr976-7tehhp7awofutrgeji7rqzctqs4ccp0v.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6169aa8a5c5d65acc5dc6b02

Commit:
fix(STN-891): fix class on global message alert (#980)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr976-7tehhp7awofutrgeji7rqzctqs4ccp0v.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6169aa8a5c5d65acc5dc6b02

Commit:
Feat(STN-882):  remove icon feature flag (#979)

* feat(STN-882): Vertical Cards no longer require experimental feature toggle

* fix: removed extra padding after icons for location and speaker to allow
text to accurately align.

* refactor: changing location/calendar/spearker icons to have the same
viewbox regardless of the actual size of the icon itself.

* chore: compile styles

* update theme settings as the icons are no longer behind feature flags

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr976-7tehhp7awofutrgeji7rqzctqs4ccp0v.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6169aa8a5c5d65acc5dc6b02

Commit:
fix(STN-875): default images not loading (#983)

* fix(STN-875): default images not loading

* chore: compile css
